names:
  - normal
  - blink
  - surprised
  - sleepy
  - angry
  - sad
  - happy
normal:
  upperlid_position: [9]
blink:
  upperlid_position: [9, 9, 130, 130, 9, 9]
surprised:
  path_iris: "/iris_surprised.jpg"
  path_pupil: "/pupil_surprised.jpg"
  path_reflex: "/reflex_surprised.jpg"
  upperlid_position: [9, 9, 9, 9, 130, 130, 9, 9, 9, 9, 130, 9, 9, 130, 9, 9]
  upperlid_default_pos_y: 8
sleepy:
  upperlid_position: [60, 60, 70, 130, 130, 130, 70, 60, 60, 60]
  upperlid_default_pos_y: 8
angry:
  upperlid_default_pos_x: 50
  upperlid_default_pos_y: 40
  upperlid_default_theta: -30
sad:
  upperlid_default_pos_x: 100
  upperlid_default_pos_y: 40
  upperlid_default_theta: 30
happy:
  path_iris: "/white.jpg"
  path_pupil: "/white.jpg"
  path_reflex: "/white.jpg"
  path_upperlid: "/reflex_happy.jpg"
  upperlid_position: [130, 131, 132, 133, 134, 135]
